ofborg/src/tasks/eval/nixpkgs.rs

```diff
@@ -270,6 +272,22 @@ impl<'a> NixpkgsStrategy<'a> {
                 },
             );
 
+            if changed_paths.len() > MAINTAINER_REVIEW_MAX_CHANGED_PATHS {
+                info!(
+                    "pull request has {} changed files, skipping review requests",
```

Comment:
Nit; might as well stay consistent with the variable's name.

```suggestion
                    "pull request has {} changed paths, skipping review requests",
```